PCPTool.v11/inc/TpmAtt.h

```diff
@@ -172,125 +172,125 @@ typedef struct _PCP_KEY_ATTESTATION_BLOB {
 
 // SIPA event structures
 
+////
```

Comment:
Please delete old declarations or code (to see what the previous code looked like, one can use git history).

Reply:
Done!